master/pkg/searcher/grid_test.go

```diff
@@ -44,17 +46,34 @@ func TestGridFunctionality(t *testing.T) {
 
 func TestHyperparameterGridMethod(t *testing.T) {
 	dParam := expconf.DoubleHyperparameter{RawMaxval: 2.0, RawCount: ptrs.IntPtr(5)}
-	assert.Equal(t, len(grid(expconf.Hyperparameter{RawDoubleHyperparameter: &dParam})), 5)
+	assert.Equal(t,
+		len(getGridAxes([]string{"x"}, expconf.Hyperparameter{RawDoubleHyperparameter: &dParam})),
```

Comment:
need to index 0 on the result of getGridAxes for this assert as well as others in this test.

Reply:
makes sense, thank you